examples/async/src/index.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { render } from 'react-dom';
-import { createStore, applyMiddleware } from 'redux';
-import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
-import { createLogger } from 'redux-logger';
-import reducer from './reducers';
-import App from './containers/App';
 
-const middleware = [thunk];
-if (process.env.NODE_ENV !== 'production') {
-  middleware.push(createLogger());
-}
+import { ApolloClient } from 'apollo-client';
+import { withClientState } from 'apollo-link-state';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloProvider } from 'react-apollo';
+
+import App from './containers/App';
+import schema from './schema';
```

Comment:
Would it be more accurate to call this a resolver map?

Reply:
probably!